package.json

```diff
@@ -6,31 +6,38 @@
   },
   "main": "src/index.js",
   "scripts": {
-    "start": "node src/index.js",
-    "watch": "nodemon src/index.js",
-    "lint": "./node_modules/.bin/eslint **/*.js"
+    "react": "cd react/ && npm start",
+    "react_build": "cd react/ && npm run build",
+    "install_all": "npm install && cd react/ && npm install && npm install --no-shrinkwrap",
+    "watch": "concurrently -n \"React,Server\" -c \"bgBlue.bold,bgMagenta.bold\" \"npm run react\" \"nodemon server\"",
+    "lint": "./node_modules/.bin/eslint **/*.js",
+    "heroku-postbuild": "cd react/ && npm install && npm install --only=dev --no-shrinkwrap && npm run build"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Martturi/cv_master.git"
   },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "concurrently": "^3.5.1",
     "dotenv": "^4.0.0",
     "ejs": "2.5.6",
     "express": "4.15.2",
     "pg": "7.4.0",
-    "promise": "^8.0.1"
+    "promise": "^8.0.1",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-scripts": "1.0.17",
+    "eslint": "^4.14.0",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-plugin-ejs": "0.0.2",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.5.1"
   },
   "license": "MIT",
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^4.12.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-ejs": "0.0.2",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
```

Comment:
Versio vaihdettu vanhempaan, huomattu eri plug-inien haluavan eri versioita josta seuraa konflikteja

Reply:
En oo tosiaan varma miten helppo tää on korjaa, varmaan pitäis vaihtaa jotain dependencejä. Voin vielä kattoa illemmalla.